AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -144,9 +162,18 @@ - (void)_calculateSize
   if (isinf(_constrainedSize.width) == NO && [_attributes.pointSizeScaleFactors count] > 0) {
     _currentScaleFactor = [[self fontSizeAdjuster] scaleFactor];
   }
-  
-  __block NSTextStorage *scaledTextStorage = nil;
+
+  // If we do not scale, do exclusion, or do custom truncation, we should just use TextKit for a fast-path.
   BOOL isScaled = [self isScaled];
```

Comment:
Not new with this diff, but should this be scalingEnabled or requiresScaling?  isScaled sounds like it already knows it will have to be scaled down.